packages/chevrotain/src/parse/parser/traits/recoverable.ts

```diff
@@ -82,6 +82,10 @@ export class Recoverable {
     return true
   }
 
+  public canTokenTypeBeDeletedInRecovery(tokType: TokenType) {
```

Comment:
add explicit return type for this function please

Reply:
Ok, should we also add a explicit return type to `canTokenTypeBeInsertedInRecovery` then (maybe in an additional PR)?

Reply:
That would be a welcome improvement as well. 👍